### DISCUSSION TITLE
Weight of iCub V 2.5 (e.g. iCubGenova02 - Purpley)

### DISCUSSION THREAD

**Discussion by DanielePucci** [robotology/community#56] [Show and tell]
Does anyone know the weight of iCub V 2.5 (e.g. iCubGenova02 - Purpley) with and without the battery? In the case of no battery, it would be nice to have the robot's weight with the dummy.

**Comment by DanielePucci** [comment]
Yes, I know.

The weight of the robot without the battery and with the dummy is 

```
32.6 Kg
```

![2015-10-07 11 52 02](https://cloud.githubusercontent.com/assets/5045846/10335066/dd46be12-6cee-11e5-9e16-54b8b986e393.jpg)

The weight of the dummy is 

```
1.3 Kg
```

![2015-10-07 11 59 16](https://cloud.githubusercontent.com/assets/5045846/10335095/fbe07ee4-6cee-11e5-8ee6-91c3603c7dda.jpg)

while the weight of the battery is 

```
1.7 Kg
```

![2015-10-07 11 59 45](https://cloud.githubusercontent.com/assets/5045846/10335103/09abd032-6cef-11e5-8321-b974cb7bd7cb.jpg)

So, the weight of the robot with the battery is:

```
33.1 Kg
```

**Comment by DanielePucci** [comment]
@iron76 @francesco-romano @traversaro @maggia80

**Comment by traversaro** [comment]
For the sake of completeness, did you check the weight of the soles ? I would expect they are under 100 grams and can be assumed as negligible, but it is always nice to check.

**Comment by DanielePucci** [comment]
Well, the soles' weight is 

```
0.2 Kg
```

![2015-10-07 13 46 00](https://cloud.githubusercontent.com/assets/5045846/10336584/05b8c45c-6cfa-11e5-8e10-a541c4473fd1.jpg)

So, in view of the floor approximation for the above weights, the robot's weight with the battery and the soles is

```
33.3 Kg
```

**Comment by pattacini** [comment]
As usual, @randaz81 always shows up in the pictures we take :smile:

**Comment by DanielePucci** [comment]
Really focused this time :+1:

**Comment by DanielePucci** [comment]
Additional pictures on the weight of single components.
![2015-10-09 10 17 37](https://cloud.githubusercontent.com/assets/5045846/10390208/6a3cdb6a-6e77-11e5-848a-60d4d95afe7a.jpg)
![2015-10-09 10 11 05](https://cloud.githubusercontent.com/assets/5045846/10390209/6a3cecfe-6e77-11e5-8e7c-67ab4559b472.jpg)
![2015-10-09 09 58 30](https://cloud.githubusercontent.com/assets/5045846/10390205/6a3bf812-6e77-11e5-949c-cdbd47c85abb.jpg)
![2015-10-09 09 43 29](https://cloud.githubusercontent.com/assets/5045846/10390207/6a3c6b3a-6e77-11e5-80a9-df29d15d3caf.jpg)
![2015-10-09 09 43 25](https://cloud.githubusercontent.com/assets/5045846/10390206/6a3c2288-6e77-11e5-9744-e5c849e75a3a.jpg)
![2015-10-09 09 42 58](https://cloud.githubusercontent.com/assets/5045846/10390204/6a3a719a-6e77-11e5-9c2f-fd148f94efac.jpg)
![2015-10-09 09 42 51](https://cloud.githubusercontent.com/assets/5045846/10390211/6a500c80-6e77-11e5-88df-c0a2a0826131.jpg)
![2015-10-09 09 34 45](https://cloud.githubusercontent.com/assets/5045846/10390212/6a522fa6-6e77-11e5-9e35-7565e102346e.jpg)
![2015-10-09 09 32 50](https://cloud.githubusercontent.com/assets/5045846/10390213/6a52b070-6e77-11e5-81aa-473832f65ca2.jpg)

**Comment by DanielePucci** [comment]
Shoulder-arm groups:
![2015-10-27 10 37 30 hdr](https://cloud.githubusercontent.com/assets/5045846/10754366/57620e42-7c97-11e5-9a58-422197f01daf.jpg)
![2015-10-27 10 37 13 hdr](https://cloud.githubusercontent.com/assets/5045846/10754365/5761f344-7c97-11e5-8d35-8e0fde829678.jpg)

@traversaro @francesco-romano @Mick3Lozzo

**Comment by DanielePucci** [comment]
@spada1978

**Comment by spada1978** [comment]
Lowerbody mechanics
weight 15.15 Kg
![20151119_142112](https://cloud.githubusercontent.com/assets/14175004/11272273/f0a154f0-8ecb-11e5-824e-1b43a5be1ca5.jpg)
![20151119_142107](https://cloud.githubusercontent.com/assets/14175004/11272277/fb43ba10-8ecb-11e5-9761-cecf1a9fa1a0.jpg)
![20151119_142103](https://cloud.githubusercontent.com/assets/14175004/11272282/02e4cf70-8ecc-11e5-9f68-28a432b46e0a.jpg)

**Comment by spada1978** [comment]
Lowerbody mechanics / electronics
weight 16.44 Kg
![20151119_145233](https://cloud.githubusercontent.com/assets/14175004/11272546/fbb45048-8ecd-11e5-8d35-4fa5c4adda95.jpg)

![20151119_145238](https://cloud.githubusercontent.com/assets/14175004/11272555/098357aa-8ece-11e5-89b4-73e16f53ec8d.jpg)
![20151119_145228](https://cloud.githubusercontent.com/assets/14175004/11272561/114cc386-8ece-11e5-8dba-25262b3c21ba.jpg)

**Comment by randaz81** [comment]
The dummy weight has been updated and now is identical to the battery weight (1.7Kg). Now installed on iCubGenova02. @DanielePucci @francesco-romano @traversaro
